### PR TITLE
feat: Persistent theme settings

### PR DESCRIPTION
**Pull request by going-confetti**
## How to Test
* Try changing the default setting
* Close the app
* Open it again - the setting should persist

## Checklist

- [ ] I have performed a self-review of my code.
- [ ] I have added tests for my changes.
- [ ] I have run linter locally (`npm run lint`) and all checks pass.
- [ ] I have run tests locally (`npm test`) and all tests pass.
- [ ] I have commented on my code, particularly in hard-to-understand areas.

## Screenshots (if appropriate):
image

### CODE REVIEW

File: src/components/ThemeSwitcher.tsx
Comment (going-confetti) [inline src/components/ThemeSwitcher.tsx:1]: Goodbye 😭

**Comment by going-confetti** [comment]
> Oh man, I loved that switch in activity bar 😞 It removes the friction to test both themes when developing, can we keep it in activity bar additionally to settings? We could also wrap in dev mode condition if there's a reason we don't want to have it there in prod builds.

Let's make it dev-only for now 👍

File: src/components/ThemeSwitcher.tsx [hunk 1]
```diff
@@ -4,6 +4,12 @@ import { IconButton, Tooltip } from '@radix-ui/themes'
 
 export function ThemeSwitcher() {
   const theme = useTheme()
+  // @ts-expect-error we have commonjs set as module option
+  const isDev = import.meta.env.DEV
```
Comment (cristianoventura) [inline src/components/ThemeSwitcher.tsx:8]: Would `process.env.NODE_ENV === 'development'` work here?

**Review by cristianoventura — DISMISSED** [review]
LGTM! 🙌

File: src/components/ThemeSwitcher.tsx (hunk 1 above)
Comment (going-confetti) [reply src/components/ThemeSwitcher.tsx]: Thanks for the suggestion. I was under the impression it wasn't available in the render process 👀. Replaced with `process.env.NODE_ENV === 'development'` 👍

**Review by cristianoventura — APPROVED** [review]

**Pull request merged**